main.ts

```diff
@@ -1,8 +1,50 @@
 import { Hono } from 'hono';
 import { accounts } from './pkg/accounts/mod.ts';
+import {apiReference} from "scalar/hono-api-reference"
 
 const app = new Hono();
 
-app.route('/accounts', accounts);
+app.route('/', accounts);
+app.get("/notes", (c) => {
+  return c.text("Hello, World!");
+})
+
+app.get("/doc.json", async (c) => {
+  const modulePath: Array<string> = ["accounts"];
```

Comment:
```suggestion
  const modulePath: string[] = ["accounts"];
```

`Array<string>` は使わないでください.

https://github.com/pulsate-dev/pulsate/blob/main/CONTRIBUTING.md#arrays